src/tests/verification_real_world/update_valid_ee_certs.rs

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env cargo +nightly -Z script
+```cargo
+package.edition = "2021"
+dependencies.anyhow = "1"
+dependencies.reqwest.version = "0.11"
+dependencies.reqwest.features = ["blocking"]
```

Comment:
Will this example work if we also specify the rustls-tls and webpki-roots features, or is it dependent on native-tls?

Reply:
5ce3fefa2c7e796a52d6a130812d1c2682d8b1a9 should do it

Reply:
Awesome :+1: